src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { TodoItem } from '../TodoItem';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
```

Comment:
You should hide the list and footer if there are no todos yet, as mentioned in the task description.